x-pack/test/alerting_api_integration/spaces_only/tests/alerting/group1/get_alert_summary.ts

```diff
@@ -308,6 +308,9 @@ export default function createGetAlertSummaryTests({ getService }: FtrProviderCo
           true
         );
 
+        // wait so cache expires
+        await setTimeoutAsync(10000);
```

Comment:
Feels like we should use a common constant instead of literal.  Will be easier to fix later :-)

Reply:
Added const in https://github.com/elastic/kibana/pull/192573/commits/883d9094e9c93cb18b493f1d91c80c6953bd2969